### ISSUE TITLE
newer version of cl-alexandria available in Ubuntu 14.04

### ISSUE THREAD

**Issue by airballking** [cram-code/cram_3rdparty#5]
Just saw: As of Ubuntu 14.04 there is a much newer version of cl-alexandria available. Do we wanna use that as of Indigo?

**Comment by gaya-** [comment]
Do you want to have it as a system dependency? In that case the following questions should be cleared first, I'd say:

* Who is behind the Debian packages? Are those the people who also develop the library? If not, is the person going to keep supporting it for future Ubuntu versions or was it a one time thing.
* Will there be any substantial API breakage?
* Will we have to take care of ASDF finding Alexandria manually after installing it from source or not?

If the answer to the last question is "yes, we'll have to do it manually", then I wouldn't switch to the Debian package and would just update the source code in our wrapper.

**Comment by phyrog** [comment]
I created a PR #7 that includes the newest tagged (if not too old) version of the libraries. alexandria is also included with the most recent commit of 2014/07/25.

**Comment by gaya-** [comment]
Found this related issue from 2013: https://github.com/cram-code/cram_core/issues/3
I guess with updating the sources in our repo and taking into account Lorenz's comment this issue could be closed. IMHO.

**Comment by airballking** [comment]
Yeah, the naive part of me was wondering whether the state of the Common LISP debians has improved in the 2 years since. I agree, with @phyrog's update to our repos that is no longer relevant.
